### PR TITLE
.github/workflows: skip unit and e2e tests for *.md file updates

### PR DESCRIPTION
**Pull request by slashpai**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

**Skip unit test and e2e test for \*.md update**

Ref:

- https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore
- https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Review by nicolastakashi — APPROVED** [review]

**Comment by ArthurSens** [comment]
We run mdox for format checking on those docs, don't we?

**Comment by slashpai** [comment]
> We run mdox for format checking on those docs, don't we?

Ya you are right just noticed that. I was trying to avoid running all tests for a simple doc change.

I think we might be able to use https://github.com/dorny/paths-filter for running specific job based on filter.

**Comment by nicolastakashi** [comment]
We can move the mdocs for a dedicated workflow and have something like that filtering by path.
https://github.com/coralogix/coralogix-operator/blob/main/.github/workflows/helm-chart-deploy.yml#L8
Or file extension.

**Comment by slashpai** [comment]
I updated the PR to ignore running unit test and e2e tests which consumes lot of time when updates are done to *.md

File: .github/workflows/e2e.yaml [hunk 1]
```diff
@@ -1,13 +1,17 @@
 name: e2e
 on:
   pull_request:
+    paths-ignore:
+      - '**/*.md'
```
Comment (slashpai) [inline .github/workflows/e2e.yaml:5]: ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

**Comment by slashpai** [comment]
@ArthurSens @nicolastakashi @simonpasquier could you please take a look

**Review by simonpasquier — APPROVED** [review]

**Comment by simonpasquier** [comment]
Thanks!

**Pull request merged**